### DISCUSSION TITLE
Cli module creation produces files which do not follow the enable position parameter

### DISCUSSION THREAD

**Discussion by Tofandel** [area17/twill#2468] [Q&A]
## Description

When creating a module or a capsule using the cli utils, we are asked if we want to use the position feature, even when we say no, the Repository file contains `$this->enableReorder();` and the migration file contains `$table->int('position')`, the model contains `hasPosition`

### Steps to reproduce

Run `php artisan twill:make:module -B -S -M -F -R -N Seminar`
Select `no` for every questions
Look at the generated files

### Expected result

No position related code

### Actual result

The position feature is completely enabled

### Versions

Twill version:  3.0.2

**Comment by ifox** [comment]
Hi @Tofandel you are using the -N flag, which stands for nested, and it needs position management.

**Comment by ifox** [comment]
I totally hear your observation though, and I think the position question should not be asked when the nested flag is passed to the CLI.

**Comment by Tofandel** [comment]
I see thanks, yes I was going to say the same, if the option is ignored with `nested` enabled the question shouldn't be asked
